managed/services/grafana/client.go

```diff
@@ -138,7 +141,7 @@ func (c *Client) do(ctx context.Context, method, path, rawQuery string, headers
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	if resp.StatusCode != 200 && resp.StatusCode != 202 {
+	if resp.StatusCode != 200 && resp.StatusCode != 201 && resp.StatusCode != 202 {
```

Comment:
let's simplify by checking if status code < 200 and > 202

Reply:
Done. I used < 200 || > 202 since it should be little faster than < 200 && > 202